server/views/pages/functionalSkills/macros/sortableTable.njk

```diff
@@ -0,0 +1,29 @@
+{#  -------------------------------------------------------------
+    Sortable table for the functional skills page
+    `config` argument is an object with the following properties:
+      * id - the id of the sortable table
+      * assessmentType - the name of the data array to loop through { englishSkills, mathsSkills, digitalSkills }
+    ------------------------------------------------------------- #}
+{% macro sortableTable(config) %}
+    <table class="govuk-table" data-module="moj-sortable-table" data-qa="{{ config.id }}-sortable-table">
+      <thead class="govuk-table__head">
+        <tr class="govuk-table__row">
+          <th scope="col" class="govuk-table__header">Name</th>
```

Comment:
Is it possible to fix the table column widths?
I notice in your first screenshot that all 3 tables dont all have the same column widths, and I'd wager some good money that we'll be asked to do this!
I vaguely remember we made a change a while back to a table/div somewhere to make the columns a consistent/fixed width? Can we do something similar here? (without adding massive delay by asking the design team what widths they want! 🙄  - can we just go with something like 20%, 40%, 20%, 20% ? )

Reply:
Fixed the column widths for Name and Grade - updated screenshot